components/python/pyparsing/Makefile

```diff
@@ -59,7 +85,14 @@ COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
 
 ASLR_MODE = $(ASLR_NOT_APPLICABLE)
 
+# Manually added dependencies
+REQUIRED_PACKAGES += library/python/pep517-37
```

Comment:
You have missed the dependencies on
library/python/build-37
library/python/build-39